npm/builder.js

```diff
@@ -49,7 +51,18 @@ Builder.prototype.compile = function (callback) {
   var opalCompiler = new OpalCompiler({dynamicRequireLevel: 'ignore'});
   opalCompiler.compile('asciidoctor-revealjs', 'build/asciidoctor-revealjs.js', ['lib']);
   typeof callback === 'function' && callback();
-}
+};
+
+Builder.prototype.replaceUnsupportedFeatures = function (callback) {
```

Comment:
Maybe add a comment to state that this is hack that need to be removed once issue XX is resolved ?

Reply:
This is already not needed, see my comments.

Reply:
I just removed it and confirmed things reach the same `ThreadSafe::Cache` error without it. It was reintroduced in the rebase process. The rebase was quite complicated to merge since that branch started before doctest was in and master was pulled in at some point.